catalogs/Dockerfile

```diff
@@ -21,4 +20,6 @@ RUN pip install -r /tmp/requirements-pip.txt
 
 WORKDIR /opt/catalogs
 
+COPY ./main.sh main.sh
+
```

Comment:
Locally, this usually gets bind mounted in by Docker Compose along with the rest of the repo. This is how the available catalogs get mounted in too. I think it's a good idea to include the entrypoint script explicitly in the image, though. It's required to run anything, whereas you only need to include the catalogs you're currently working with.

Reply:
Just worth surfacing if this file isn't mentioned in the `.dockerignore` file as well, it will always be overwritten by the bind mount. So, it'll only be used invoking the container image without the volume mount.

Reply:
Yup! Currently there aren't many cases where we would invoke this image without bind mounting the whole repo in. I can see some general utility here if the image were published to ECR, pulled down, and a catalog that isn't tracked in the main repo was mounted in. Or if we baked catalogs into the image or into a derivative image at a later date.

Reply:
Oh yeah, I agree. I think it _should_ be copied into the image. I just wanted to document the behavior.